### PR TITLE
Update install-md-app-guard.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation in the steps to install by using Intune

fixes #https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10770

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: windows/security/threat-protection/microsoft-defender-application-guard/install-md-app-guard.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Application Guard functionality is turned off by default. However, you can quick
 
    1. In the **Platform** list, select **Windows 10 and later**. 
    
-   1. In the **Profile** list, select **Endpoint protection**. 
+   1. In the **Profile** Type, Choose **Templates** and select **Endpoint protection** .
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-application-guard/install-md-app-guard.md:103]:
```suggestion
   1. In the **Profile** type, choose **Templates** and select **Endpoint protection**. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Pull request merged**